docs/source/sensors/Apollo.rst

```diff
@@ -0,0 +1,2 @@
+Apollo Metric
+=============
```

Comment:
Just confirming: The intention of the PR is to get the structure and ToC in, not to also include content in these?

Reply:
Correct, it's mainly so users see the following on the main page:
<img width="1344" alt="Screenshot 2024-03-08 at 11 55 51 AM" src="https://github.com/DOI-USGS/usgscsm/assets/17728237/1ef2585d-10c7-4a7d-8721-b8fcfb2274fb">